docs/_redirects

```diff
@@ -0,0 +1 @@
+/ https://www.algolia.com/doc/guides/building-search-ui/what-is-instantsearch/vue/
```

Comment:
I think we talked about this but maybe I misunderstood it.
Shouldn't there be a redirect for all the other pages:
example
-  `/getting-started/installing.html` -> corresponding page in the new docs
- `/getting-started/using-components.html` -> corresponding page in the new docs
...

Reply:
Not for Vue ([see reference](https://docs.google.com/document/d/1B-eWm7WJapqE5fMmI8HwsT_hZF_z6s0A0QWyA08SxNk/edit#heading=h.fcxqldkv871a)).